about.html

```diff
@@ -59,15 +94,18 @@ <h1 class="headline-about">NEO STOICISM </br> SUMMIT 2023</h1>
 
 
   <footer class="footer-about">
-    <div class="footer-left">
-      <img src="images/cc.png" alt="Logo">
-    </div>
-    <div class="footer-right">
-      <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+    <div class="footer-container">
```

Comment:
- [optional] you are missing the partners section in the mobile version of the about page. Kindly check that